### PR TITLE
feat: Upgrade v8.10 to loaders.gl v4

### PR DESCRIPTION
**Pull request by ibgreen**
#### Background
- There are a number of features in loaders.gl 4.0 is needed by of our products, and having 8.10 alphas that are based on 4.0 will make early integration possible.
- Bumping on 8.10 branch also means that we will get early test mileage on math.gl 4.0 and loaders.gl 4.0. Among other things we will be able to bump kepler.gl on a branch.
#### Change List
- Bump to loaders.gl@4.0.0-beta.6

### CODE REVIEW

**Comment by Pessimistress** [comment]
Unless we are only ever releasing 8.10 for interna testing, it should stay on v3.

**Comment by ibgreen** [comment]
> Unless we are only ever releasing 8.10 for internal testing

- There are a number of features in loaders.gl 4.0 is needed by of our products, and having 8.10 alphas that support are based on 4.0 will make early integration possible.
- Bumping on 8.10 branch also means that we will get early test mileage on math.gl 4.0 and loaders.gl 4.0. Among other things we will be able to bump kepler.gl on a branch.

File: package.json [hunk 1]
```diff
@@ -80,5 +80,9 @@
   "engines": {
     "node": ">=14"
   },
-  "dependencies": {}
+  "dependencies": {},
+  "volta": {
```
Comment (Pessimistress) [inline package.json:84]: Can you split this into a different PR

**Review by Pessimistress — APPROVED** [review]

**Comment by ibgreen** [comment]
@felixpalmer @igorDykhta FYI - Merged, but still need to publish a new 8.10 alpha.

**Comment by ibgreen** [comment]
8.10.0-alpha.7
